pkg/diff/diff.go

```diff
@@ -582,7 +582,7 @@ func generateDiffString(e crud.Event, isDelete bool, noMaskValues bool) (string,
 }
 
 // Solve generates a diff and walks the graph.
-func (sc *Syncer) Solve(ctx context.Context, parallelism int, dry bool, isJSONOut bool) (Stats,
+func (sc *Syncer) Solve(ctx context.Context, parallelism int, dry bool, isJSONOut bool, noDeletes bool) (Stats,
```

Comment:
If possible we'd make a similar change as in https://github.com/Kong/deck/commit/152977482599301b8559d3c7f70801bcb5222a3a to not add yet another `bool` parameter here.